qabelbox/src/main/java/de/qabel/qabelbox/providers/DocumentIdParser.java

```diff
@@ -30,7 +30,7 @@ public String getPrefix(String documentId) throws FileNotFoundException {
 
 
     public String getFilePath(String documentId) throws FileNotFoundException {
-        String[] split = documentId.split(BoxProvider.DOCID_SEPARATOR, 4);
+        String[] split = documentId.split(BoxProvider.DOCID_SEPARATOR, 3);
```

Comment:
Why did you only change the number of splits in this method and not the other methods? Do the other methods work with filenames that contain the DOCID_SEPERATOR?